src/dev-box/dev-box.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: apps/v1
+kind: DaemonSet
```

Comment:
Suggest to rename this file to dev-box-k8s-deploy.yaml

Reply:
updated